src/groups/special_euclidean.jl

```diff
@@ -713,18 +713,52 @@ function vee(M::SpecialEuclidean, p::ArrayPartition, X::ArrayPartition)
     M1, M2 = M.manifold.manifolds
     return vcat(vee(M1.manifold, p.x[1], X.x[1]), vee(M2.manifold, p.x[2], X.x[2]))
 end
-function hat(M::SpecialEuclidean{2}, p::ArrayPartition, c::SVector)
+function get_coordinates(
+    M::SpecialEuclidean,
+    p::ArrayPartition,
+    X::ArrayPartition,
+    basis::AbstractBasis,
```

Comment:
I'm trying to fix this but I'm not sure what call this is supposed to handle. In general, you should try to use as narrow basis types as possible here. In particular. not all bases can be propagated like in this method (for example `CachedBasis`). Please try using either of these here:
- `DefaultOrthonormalBasis`
- `DefaultOrthogonalBasis`
- `VeeOrthogonalBasis`,
and similarly in `get_vector`.

Reply:
Thanks for the tip, I went with `DefaultOrthogonalBasis`. but I'm not sure what the difference between `VeeOrthogonalBasis `and `DefaultOrthogonalBasis` is.

Reply:
There isn't much of a difference currently, and I'm not sure we even need `DefaultOrthogonalBasis` for anything. If you like using `vee`/`hat` then it's probably better to just use `VeeOrthogonalBasis`.

Reply:
To better follow the Manifolds.jl standard, we were moving away from vee/hat to using `get_coordinates/vector` with `DefaultOrthogonalBasis` (I didn't know about `VeeOrthoganolBasis`) so it's used in a few places. We currently only support Lie Groups, but the long-term plan is to support all manifolds in Manifolds.jl.